test/Microsoft.Azure.Devices.Edge.Test/helpers/X509ManualProvisioningFixture.cs

```diff
@@ -88,7 +88,8 @@ async Task<(X509Thumbprint, IdCertificates)> CreateIdentityCertAsync(string devi
 
             return (new X509Thumbprint()
             {
-                PrimaryThumbprint = deviceCert.Thumbprint
+                PrimaryThumbprint = deviceCert.Thumbprint,
+                SecondaryThumbprint = deviceCert.Thumbprint
```

Comment:
Should the two thumbprints be different? or can it be whatever?

Reply:
they can be whatever, as long as one of them matches the client cert